libs/auth/scopes/src/lib/api.scope.ts

```diff
@@ -9,4 +9,5 @@ export enum ApiScope {
   licenses = '@island.is/licenses',
   licensesVerify = '@island.is/licenses:verify',
   company = '@island.is/company',
+  vehicles = '@island.is/internal',
```

Comment:
Why changing to internal?

Reply:
omg, great catch. I was debugging and changed it, and forgot to revert 😞

Reply:
Should be all fixed up now. ✅